app/config/locale/templates/email-base-styled.tpl

```diff
@@ -85,50 +94,14 @@
                 margin-right: 24px;
                 margin-top: 8px;
             }
-            .main a.button:hover,
-            .main a.button:focus {
+            a.button:hover,
+            a.button:focus {
                 opacity: 0.8;
             }
-            .main a.button.is-reverse {
-                background: #fff;
-                color: #fd366e;
-            }
-            .fs12 {
-                font-size: 12px;
-            }
-            .ta-right {
-                text-align: right !important;
-            }
-            .ta-left {
-                text-align: left !important;
-            }
-            .ff-dmsans {
-                font-family: "DM Sans", sans-serif !important;
-            }
-            .ff-inter {
-                font-family: "Inter", sans-serif !important;
-            }
-            .w500 {
-                font-weight: 500 !important;
-            }
-            .w400 {
-                font-weight: 400 !important;
-            }
-            .w600 {
-                font-weight: 600 !important;
-            }
-            .tc-accent {
-                color: #da1a5b;
-            }
-            .pt-16 {
-                padding-top: 16px !important;
-            }
-            .pt-32 {
-                padding-top: 32px !important;
-            }
-            .divider {
-                padding-top: 32px;
-                border-bottom: solid 1px #d8d8db;
```

Comment:
Those are some utility classes, I didnt see them used anywhere.
Since I was simplifying and giving structure to those base email templates, I decided to remove them.

I did QA and all emails look fine.